FreeRTOS-Plus/Demo/coreSNTP_Windows_Simulator/SNTPClientTask.c

```diff
@@ -686,13 +686,13 @@ void calculateCurrentTime( UTCTime_t * pBaseTime,
     /* Set the current UTC time in the output parameter. */
     if( msElapsedSinceLastSync >= 1000 )
     {
-        pCurrentTime->secs = pBaseTime->secs + msElapsedSinceLastSync / 1000;
+        pCurrentTime->secs = ( uint32_t ) ( pBaseTime->secs + msElapsedSinceLastSync / 1000 );
```

Comment:
Wondering if overflow can happen here. If it is possible, we should handle it here as well.

Reply:
By overflow , do you mean since `msElapsedSinceLastSync` is `uint64_t` ?
We can have something like this then :
```
if ( msElapsedSinceLastSync > UINT32_MAX )
{
      msElapsedSinceLastSync = msElapsedSinceLastSync - UINT32_MAX;
}
```